src/shared/approval-status/get-approval-status.js

```diff
@@ -37,7 +38,21 @@ const getApprovalStatusIcon = approvalStatus => {
     }
 }
 
-const getApprovalStatusText = approvalStatus => {
+const getApprovedStatusText = ({ approvedAt, approvedBy }) => {
+    const approvedMsg = approvedBy
+        ? i18n.t('Approved by {{approvedBy}}', { approvedBy })
+        : i18n.t('Approved')
+
+    if (!approvedAt) {
+        return approvedMsg
+    }
+
+    const approvedAtMsg = moment(approvedAt).fromNow()
+
+    return `${approvedMsg} ${approvedAtMsg}`
```

Comment:
In order to support right to left languages, `approvedAtMsg` needs to be interpolated using `i18n.t`